### ISSUE TITLE
SMTP with auth

### ISSUE THREAD

**Issue by MrGecco** [Alanaktion/phproject#311]
Would be great if it is possible to send mails via SMTP with auth.
The php mail() function isn't appropriate for that. Furthermore, in some shared hosting environments the function isn't working reliable.

**Comment by Alanaktion** [comment]
I've worked around this by using a local mail relay like postfix, but this is definitely something that would be helpful for shared hosting environments.

**Comment by bestkiller92** [comment]
I agree with this idea. Indeed this could be very usefull. Could you please keep me in touch if you start developping on this ?
